library/math/numbersLine/src/lib/utils.ts

```diff
@@ -7,9 +7,11 @@ export function cn(...inputs: ClassValue[]) {
 }
 
 export const calculationWidthScreen = (windowWidth: number)=>{
-  return -windowWidth*((LineRange.hundred-RulerLenth.hundred)/RulerLenth.hundred)
+  return -1*windowWidth*((LineRange.hundred-RulerLenth.hundred)/RulerLenth.hundred)
```

Comment:
why -1 and not -

Reply:
i think it is more clear 
but i change if you think it is not

Reply:
ok.